drake/solvers/Constraint.h

```diff
@@ -39,24 +39,24 @@ class Constraint {
   Constraint(size_t num_constraints, Eigen::MatrixBase<DerivedLB> const& lb,
              Eigen::MatrixBase<DerivedUB> const& ub)
       : lower_bound(lb), upper_bound(ub) {
-    check(num_constraints);
+    Check(num_constraints);
   }
   virtual ~Constraint() {}
 
   // TODO: consider using a Ref for `y` too.  This will require the client
   // to do allocation, but also allows it to choose stack allocation instead.
-  virtual void eval(const Eigen::Ref<const Eigen::VectorXd>& x,
+  virtual void Eval(const Eigen::Ref<const Eigen::VectorXd>& x,
                     Eigen::VectorXd& y) const = 0;
-  virtual void eval(const Eigen::Ref<const TaylorVecXd>& x,
+  virtual void Eval(const Eigen::Ref<const TaylorVecXd>& x,
                     TaylorVecXd& y) const = 0;  // move this to
                                                 // DifferentiableConstraint
                                                 // derived class if/when we need
                                                 // to support non-differentiable
                                                 // functions
 
-  Eigen::VectorXd const& getLowerBound() const { return lower_bound; }
-  Eigen::VectorXd const& getUpperBound() const { return upper_bound; }
-  size_t getNumConstraints() const { return lower_bound.size(); }
+  Eigen::VectorXd const& get_lower_bound() const { return lower_bound; }
+  Eigen::VectorXd const& get_upper_bound() const { return upper_bound; }
```

Comment:
Not sure what the antecedent of "these" is, but the style guide definitely requires a trailing underscore: https://google.github.io/styleguide/cppguide.html#Variable_Names